app/server/api-generator/src/test/java/io/syndesis/server/api/generator/soap/parser/SoapApiModelParserTest.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (C) 2016 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.syndesis.server.api.generator.soap.parser;
+
+import java.io.IOException;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
+
+import io.syndesis.server.api.generator.soap.AbstractSoapExampleTest;
+import io.syndesis.server.api.generator.soap.SoapApiModelInfo;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+/**
+ * Test WSDL model parsing using {@link SoapApiModelParser}.
+ */
+@RunWith(Parameterized.class)
+public class SoapApiModelParserTest extends AbstractSoapExampleTest {
+
+    public SoapApiModelParserTest(String resource) throws IOException, InterruptedException {
+        super(resource);
+    }
+
+    @Test
+    public void parseSoapAPI() throws IOException {
+        final SoapApiModelInfo soapApiModelInfo = SoapApiModelParser.parseSoapAPI(this.specification);
+
+        assertThat(soapApiModelInfo).isNotNull();
+        assertThat(soapApiModelInfo.getModel()).isNotNull();
```

Comment:
Add proper assertions

Reply:
Added remaining notnull and notempty assertions for the rest of `soapApiModelInfo` properties. 
Checking what those values are is not necessary, since they are obtained from parsing the WSDL using javax.wsdl and CXF WSDL parsing APIs. Testing those values would turn this into a test of the underlying libraries.